libs/openchallenges/home/src/lib/statistics-viewer/statistics-viewer.component.html

```diff
@@ -2,5 +2,8 @@
   <h3 class="section-title">Total Challenges Tracked</h3>
   <div class="col container">
     <div echarts [options]="chartOptions"></div>
+    <span class="caption">
+      *The plot excludes future challenges and/or challenges with no known start date.
```

Comment:
any final suggestions on this sentence? @tschaffter @vpchung

Reply:
as @tschaffter suggests, the future/upcoming is not super accurate and might confuse with users, as there could be some upcoming challenges launching this year. 

How about: 
>*The plot does not include challenges that are planned beyond this year, or those with unspecified start dates.

Reply:
I like!  Thanks for updating the caption~

Reply:
I like the latest caption! Nicely done!